src/include/common/configs.h

```diff
@@ -87,6 +87,12 @@ struct CopyCSVConfig {
     // Size (in bytes) of the chunks to be read in InMemNode/RelCSVCopier
     static constexpr uint64_t CSV_READING_BLOCK_SIZE = 1 << 23;
 
+    // Number of tasks to be assigned in a batch when reading files.
+    static constexpr uint64_t NUM_TASKS_PER_BATCH = 200;
+
+    // Minimum number of tasks in a queue. If reached, assign a new batch.
```

Comment:
Minimum number of tasks in a queue. If reached, assign a new batch. -> Lower bound for number of incomplete tasks in copier to trigger scheduling a new batch.